stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Extensions.kt

```diff
@@ -28,28 +28,44 @@ public fun Channel.getUnreadMessagesCount(forUserId: String = ""): Int {
     }
 }
 
+@Deprecated(
+    message = "Use class member instead",
+    level = DeprecationLevel.WARNING,
+)
 public var User.image: String
-    get() = getExternalField(this, EXTRA_IMAGE)
+    get() = image
```

Comment:
These properties can now be simplified to 

```kotlin
public var User.image: String by User::image
```

Not that it matters, as they can't be invoked anymore anyway (except through Java static calls via `ContentUtils`).